search.go

```diff
@@ -606,3 +610,22 @@ func MemoryNeededForSearchResult(req *SearchRequest) uint64 {
 
 	return uint64(estimate)
 }
+
+// SetSortFunc sets the sort implementation to use when sorting hits.
+//
+// SearchRequests can specify a custom sort implementation to meet
+// their needs. For instance, by specifying a parallel sort
+// that uses all available cores.
+func (r *SearchRequest) SetSortImpl(s func(sort.Interface)) {
+	r.SortImpl = s
+}
+
+// GetSortFunc returns the sort implementation to use when sorting hits.
+// Defaults to sort.Sort.
+func (r *SearchRequest) GetSortImpl() func(data sort.Interface) {
```

Comment:
Typically in Go, the convention is not to have methods for the form GetField(), instead they use just Field() and SetField().